apps/app/app/(authed)/collectivite/[collectiviteId]/(acces-restreint)/referentiel/[referentielId]/action/[actionId]/_components/comments/action-comments.item.tsx

```diff
@@ -129,27 +128,65 @@ const ActionCommentItem = ({
 
           {/* Content */}
           <div className=" flex flex-col gap-2 w-full">
-            <div className="sm:h-7 flex sm:items-center gap-2 max-sm:flex-col">
-              <span className="text-primary-9 text-sm font-bold">
-                {comment.createdByPrenom} {comment.createdByNom}
-              </span>
-              <span className="text-grey-6 text-xs font-medium">
-                {getModifiedSince(creationDate.toISOString())}
-              </span>
-              {canUpdateOrDeleteComment && (
-                <Button
-                  className="invisible opacity-0 transition-opacity duration-150 group-hover:visible group-hover:opacity-100 focus-visible:visible focus-visible:opacity-100"
-                  dataTest="ActionDiscussionCommentaireMenu"
-                  icon="delete-bin-6-line"
-                  title="Supprimer mon commentaire"
-                  variant="outlined"
-                  size="xs"
-                  loading={isDeletingDiscussionMessage}
-                  disabled={isDeletingDiscussionMessage}
-                  onClick={() => {
-                    void handleDeleteDiscussionMessage();
-                  }}
-                />
+            <div className="sm:h-7 flex sm:items-center gap-2 max-sm:flex-col justify-between">
+              <div className="flex align-center items-center gap-2">
+                <span className="text-primary-9 text-sm font-bold">
+                  {comment.createdByPrenom} {comment.createdByNom}
+                </span>
+                <span className="text-grey-6 text-xs font-medium">
+                  {getModifiedSince(creationDate.toISOString())}
+                </span>
+                {canUpdateOrDeleteComment && (
+                  <div className="flex gap-2">
+                    <Button
+                      className="invisible opacity-0 transition-opacity duration-150 group-hover:visible group-hover:opacity-100 focus-visible:visible focus-visible:opacity-100"
```

Comment:
Globalement dans l'app et les design il y a un problème de size.
Là par exemple même si on met `size="xs"`, le bouton parrait trop gros.
Ce serait bien de revoir les paddings pour les boutons d'édition et de suppression.

```suggestion
                      className="p-1.5"
```